remoting/zookeeper/listener.go

```diff
@@ -33,6 +34,10 @@ import (
 	"github.com/apache/dubbo-go/remoting"
 )
 
+var (
+	wheel = gxtime.NewWheel(gxtime.TimeSecondDuration(ConnDelay), MaxFailTimes+1) // wheel longest span is 2 minute
```

Comment:
If you think your time span is greater than 2 minutes, you can use time.After() directly. Because the go gc will collect a minute time span timer in time. If not, you can use wheel.After.

Reply:
max is 45 seconds

Reply:
use getty.Wheel().After instead.